adminapp/src/shared/react/useLoginRedirectLink.jsx

```diff
@@ -0,0 +1,26 @@
+import useSessionStorageState from "./useSessionStorageState";
```

Comment:
Move this out of shared I think. It seems generic but in practice ends up being too specific.

Reply:
Good catch.